x-pack/plugin/runtime-fields/src/main/java/org/elasticsearch/xpack/runtimefields/mapper/BooleanFieldScript.java

```diff
@@ -84,4 +86,26 @@ public void value(boolean v) {
             script.emit(v);
         }
     }
+
+    public static class EmitValues {
```

Comment:
Can we add a comment to make it clear that this is temporary and only exposed through script-less runtime fields, undocumented on purpose?

Reply:
++